scripts/ammoBoxSpawner.sqf

```diff
@@ -42,14 +42,14 @@ _spawn_medical_box = {
   clearbackpackcargoGlobal _crate;
 
   _crate addItemCargoGlobal ["ACE_quikclot", 100]; 
-  _crate addItemCargoGlobal ["ACE_epinephrine", 20]; 
-  _crate addItemCargoGlobal ["ACE_morphine", 20]; 
-  _crate addItemCargoGlobal ["ACE_EarPlugs", 5]; 
+  _crate addItemCargoGlobal ["ACE_epinephrine", 50]; 
+  _crate addItemCargoGlobal ["ACE_morphine", 50]; 
+  _crate addItemCargoGlobal ["ACE_EarPlugs", 20]; 
   _crate addItemCargoGlobal ["ACE_surgicalKit", 4]; 
-  _crate addItemCargoGlobal ["ACE_elasticBandage", 100]; 
```

Comment:
All these values were picked so that you could carry this box and not have to drag it, as medical resupplies are likely the kind of thing you'd like to carry in. I suggest ensuring this box is still carryable.